Gruntfile.js

```diff
@@ -546,6 +556,57 @@ module.exports = function (grunt) {
                     'never',
                     '-c',
                     configFile,
+                    '--dir',
+                ],
+            },
+            (error, result, code) => {
+                if (error) {
+                    grunt.fail.fatal(
+                        `electron-builder exited with error code ${code}:\n\n${result.stdout}`,
+                        code,
+                    );
+                }
+
+                taskDoneCallback();
+            },
+        );
+    });
+
+    grunt.registerMultiTask('electron-builder-pack', function () {
+        const { dropPath } = this.data;
+        const configFile = path.join(dropPath, 'electron-builder.yml');
+
+        mustExist(configFile, 'Have you built the product you are trying to pack?');
+
+        let unpackedDirName;
+
+        switch (process.platform) {
+            case 'win32':
+                unpackedDirName = 'win-unpacked';
```

Comment:
I wasn't able to find this mapping documented on https://electron.build/ - did you identify this mapping by reverse engineering the actual `--dir` output, or is it documented somewhere? If the latter, it would be nice to cite a source

Reply:
Unfortunately, I didn't find any documentation either--why [mac doesn't get `-unpacked` appended](https://github.com/electron-userland/electron-builder/blob/7194c388f64cf9074e7ae14e74a7783da76ea284/packages/app-builder-lib/src/platformPackager.ts#L119) is a mystery to me.